src/transaction/components/TransactionSponsor.test.tsx

```diff
@@ -20,11 +20,30 @@ describe('TransactionSponsor', () => {
   });
   it('does not render if hasPaymaster is false', () => {
     (useTransactionContext as vi.Mock).mockReturnValue({
-      lifeCycleStatus: { statusName: 'transactionIdle', statusData: null },
+      lifeCycleStatus: { statusName: 'init', statusData: null },
       hasPaymaster: false,
     });
     render(<TransactionSponsor />);
 
     expect(screen.queryByText('Zero transaction fee')).not.toBeInTheDocument();
   });
+  it('does not render if statusName is not init', () => {
```

Comment:
My take on spaces with test is:
- no space within an `it`
- one space between `it`

By doing that it's much easiser reading large file full of tests.